packages/server/postgres/migrations/1685721573097_addPrePostMortemTemplates.ts

```diff
@@ -982,13 +982,12 @@ const getTemplateIllustrationUrl = (filename: string) => {
   const partialPath = `Organization/aGhostOrg/${filename}`
   if (cdnType === 'local') {
     return `/self-hosted/${partialPath}`
-  } else if (cdnType === 's3') {
```

Comment:
i don't like mutating past migrations, but this logic doesn't change anything, it just adds support for gcs & any future providers